src/Routing.tsx

```diff
@@ -75,6 +76,10 @@ const getRoutes = ({ enableServiceAccounts, isITLess, isWorkspacesFlag, isCommon
       },
     ],
   },
+  {
```

Comment:
should be a child route

Reply:
![image](https://github.com/user-attachments/assets/e2332170-04b8-49e1-a713-b3f2e3dc2fc8)
Rendering this as a child route adds the **Users and Users Groups** header above - is there a way to avoid this?